backend/tests/test_restore_consumer.py

```
@@ -0,0 +1,94 @@
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)